src/Sylius/Bundle/ThemeBundle/.gitignore

```diff
@@ -0,0 +1,5 @@
+vendor/
+bin/
+
+composer.phar
+composer.lock
```

Comment:
The same is in every Sylius bundle and component, need to discuss this issue globally.